projects/ad719x_asdz/coraz7s/system_project.tcl

```diff
@@ -1,19 +1,53 @@
 ###############################################################################
-## Copyright (C) 2022-2023 Analog Devices, Inc. All rights reserved.
+## Copyright (C) 2022-2024 Analog Devices, Inc. All rights reserved.
 ### SPDX short identifier: ADIBSD
 ###############################################################################
 
+# In the case of:
+# EVAL-AD7190 and EVAL-AD9175:
+#   it works both connecting its PMOD to PMOD JA of Cora, and
+#   placing it on top of Cora, connecting it to the Arduino header
+# EVAL-AD7193: only IO_PMOD_N=1;
+#   works only by placing it on top of Cora, connecting it to the Arduino header
+
+# make  or  make IO_PMOD_N=0 - connect the eval board PMOD to PMOD JA of Cora
+# make IO_PMOD_N=1 - connect the eval board to the Arduino header (placing it on top of Cora)
+
 source ../../../scripts/adi_env.tcl
 source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
 source $ad_hdl_dir/projects/scripts/adi_board.tcl
 
-adi_project ad719x_asdz_coraz7s
+adi_project ad719x_asdz_coraz7s 0 [list \
+  IO_PMOD_N [get_env_param IO_PMOD_N 0] \
+]
 
 adi_project_files ad719x_asdz_coraz7s [list \
     "$ad_hdl_dir/library/common/ad_iobuf.v" \
     "$ad_hdl_dir/projects/common/coraz7s/coraz7s_system_constr.xdc" \
-    "system_constr.xdc" \
-    "system_top.v"
 ]
 
+switch [get_env_param IO_PMOD_N 0] {
```

Comment:
since this chooses between the Pmod and Arduino connectors, rename "io" to "ardz". also add the build param to sysid in  system_bd.tcl (see ad4630 as example)

Reply:
Done in the last commit